### ISSUE TITLE
[receiver/k8scluster] Optionally add labels and annotations from node as resource attributes to node metrics

### ISSUE THREAD

**Issue by TylerHelmuth** [open-telemetry/opentelemetry-collector-contrib#27910]
### Component(s)

receiver/k8scluster

### Is your feature request related to a problem? Please describe.

The k8scluster collects several node-related metrics and it would be helpful to be able to add node labels and annotations as resource attributes to these metrics.

### Describe the solution you'd like

The ability to enable the receiver to add node labels and annotations as resource attributes.  The labels/annotations should be taken as is, with no manipulation applied.

### Additional context

Closely related to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/22620, but the receiver should have the capability to add the attributes itself.

**Comment by haoqixu** [comment]
I'd like to work on this.

**Comment by haoqixu** [comment]
I will submit another PR for this issue after #25134 is resovled (#29769).

**Comment by haoqixu** [comment]
remove stale
